docs/actions/ssh.md

```diff
@@ -0,0 +1,264 @@
+    |      `owner:group` | String                                                                                                     |
+
+### Example
+
+=== "Kotlin"
+``` kotlin
+``` kotlin
+SftpFileInfoTask(
+    target = "myTarget",
+    file = "/path/to/file", // (1)
+    timeout = "42 ms"
+)
+```
+
+1. The file path on the target machine
+
+## ListDir
+
+=== "Inputs"
+
+    | Required | Name          | Type                                               | Default |
+    |:--------:|:--------------|:---------------------------------------------------|:-------:|
+    |    *     | `target`      | String                                             |         |
+    |    *     | `directory`   | String                                             |         |
+    |          | `timeout`     | String ([Duration](/actions/other/#duration-type)) | `5 sec` |
+
+=== "Outputs"
+
+    |    Name | Type          |
+    |--------:|:--------------|
+    | `files` | List<String\> |
+
+### Example
+
+=== "Kotlin"
+``` kotlin
+``` kotlin
+SftpListDirTask(
+    target = "myTarget",
+    directory = "/path/to/directory", // (1)
+    timeout = "42 ms"
+)
+```
+
+1. The directory path on the target machine
+
+## Upload
+
+=== "Inputs"
+
+    | Required | Name          | Type                                               | Default |
+    |:--------:|:--------------|:---------------------------------------------------|:-------:|
+    |    *     | `target`      | String                                             |         |
+    |    *     | `source`      | String                                             |         |
+    |    *     | `destination` | String                                             |         |
+    |          | `timeout`     | String ([Duration](/actions/other/#duration-type)) | `5 sec` |
+
+SCP upload has no outputs
+
+### Example
+
+=== "Kotlin"
+``` kotlin
+``` kotlin
+SftpUploadTask(
+    target = "myTarget",
+    source = "/path/to/the/source/file", // (1)
+    destination = "/path/to/the/destination/file", // (2)
+    timeout = "42 ms"
+)
+```
+
+1. The source file path on the machine running Chutney
+2. The destination path on the target machine
+
+# SSH
+
+## Client
+
+=== "Inputs"
+
+    | Required | Name       | Type   | Default   | Values               |
+    |:--------:|:-----------|:-------|:---------:|:---------------------|
+    |    *     | `target`   | String |           |                      |
+    |    *     | `commands` | String |           |                      |
+    |          | `channel`  | String | `COMMAND` | `COMMAND` or `SHELL` |
+
+=== "Outputs"
+
+    |    Name   | Type                                   |
+    |----------:|:---------------------------------------|
+    | `results` | List<[CommandResult](#commandresult)\> |
+
+### Example
+
+=== "Kotlin"
+``` kotlin
+``` kotlin
+SshClientTask(
+    target = "myTarget",
+    commands = "[{command: "whoami", timeout:"10 s"}]", // (1)
```

Comment:
it doesnt compile, you need to escape and it need to be a  List<Any>

Reply:
done